src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensRangeHandler.cs

```diff
@@ -205,10 +226,28 @@ private void OnCompilationAvailable(Project project, Checksum projectChecksum)
                 _projectIdToLastComputedChecksum[project.Id] = projectChecksum;
             }
 
-            EnqueueSemanticTokenRefreshNotification();
+            EnqueueSemanticTokenRefreshNotification(documentUri: null);
         }
 
         private bool ChecksumIsUnchanged_NoLock(Project project, Checksum projectChecksum)
             => _projectIdToLastComputedChecksum.TryGetValue(project.Id, out var lastChecksum) && lastChecksum == projectChecksum;
+
+        public void Dispose()
```

Comment:
Felt like this method made more sense at the bottom of the file so moved it here. If anyone has opposing thoughts though lmk